pkgs/os-specific/linux/dstat/default.nix

```diff
@@ -37,5 +37,6 @@ python3Packages.buildPythonApplication rec {
     license = licenses.gpl2;
     platforms = platforms.linux;
     maintainers = with maintainers; [ ];
+    changelog = "https://raw.githubusercontent.com/dstat-real/dstat/v${version}/ChangeLog";
```

Comment:
This project is archived and has no maintainer. Probably a better idea to drop it.

Reply:
I still use this program frequently. It still works fine.

Reply:
Wonder if it could be swapped for `https://github.com/scottchiefbaker/dool`

Reply:
`dool` is not in nixpkgs. Please only remove `dstat` once `dool` is availble.

Reply:
That is something for another PR.